sample-compose/app/src/main/java/co/nimblehq/sample/compose/extensions/FlowExt.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.sample.compose.extensions
+
+import android.annotation.SuppressLint
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import kotlinx.coroutines.flow.*
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
+@SuppressLint("ComposableNaming")
```

Comment:
🙈

Reply:
It's not a Composable view, it's an extension so we should not follow the Composable naming convention 😛